projects/packages/my-jetpack/_inc/state/actions.js

```diff
@@ -256,6 +256,18 @@ const installStandalonePluginForProduct = productId => async store => {
 	} );
 };
 
+/**
+ * Request to set the welcome banner as dismissed
+ *
+ * @returns {Promise} - Promise which resolves when the banner is dismissed.
+ */
+const dismissWelcomeBanner = () => async () => {
```

Comment:
Let's also change the value in the reducer state as well? Currently, we just run side effect, but nothing changes in the app.

I understand we want to dismiss the banner on page refresh, but it shouldn't decide about data flow during runtime of the app. It's more a "view" layer decision what to do with that information that changed.